features/step_definitions/post_steps.rb

```diff
@@ -0,0 +1,47 @@
+Given(/^I create a post that says "(.*?)"$/) do |content|
+  visit "/posts/new"
+  fill_in "Content", with: Faker::Company.buzzword
+  click_button "Create Post"
+  @my_post = Post.last
+end
+
+Given(/^Another user makes a post that says "(.*?)"$/) do |content|
+  user = FactoryGirl.create(
+    :user,
+    username:"zorldo",
+    email: "zorldo@example.com",
+    password: "password"
+    )
+  @zorldo_post = FactoryGirl.create(:post, user: user, content: content)
+end
+
+When(/^I edit my post to say "(.*?)"$/) do |content|
+  visit "/posts/#{@my_post.id}/edit"
+  expect(page).to have_content("Editing Post")
+  fill_in "Content", with: content
+  click_button "Update Post"
+end
+
+When(/^I try to edit that post$/) do
+  visit "/posts/#{@zorldo_post.id}/edit"
+end
+
+When(/^I favorite that user's post$/) do
+  FactoryGirl.create(:favorite_handler, user: @current_user, post: @zorldo_post)
```

Comment:
With cucumber tests, in the `When`s, you want to do everything the way a user would do it. So here, you want to go through the page interactions a user would take, instead of calling the factory.